docs/core-concepts/accounts.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Accounts
+---
+
+# Accounts
+
+Accounts within Solana are used to store state. They are an essential
+building block for developing on Solana.
+
+## Facts
+
+::: tip Fact Sheet
+- Accounts are used to store data
+- Each account has a unique address
+- Accounts have a max size of 10MB
+- Account size is static
+- Account data storage is paid with rent
+- Default account owner is the System Program
+:::
+
+## Deep Dive
+
+### Account Model
+
+There are 3 kinds of accounts on Solana:
+
+- Data accounts store data
+- Program accounts store executable programs
+- Native accounts that indicate native programs on Solana such as System, Stake, and Vote
```

Comment:
Could add something like this:

> If you're coming from Ethereum, it's important to note that Solana separates data and code! While Ethereum contracts are stored alongside the data they maintain, Solana programs read/write data to and from independent data accounts.

Reply:
We are generally trying to not make references to Ethereum within the cookbook. We may make a guide separate to address the differences, but leave out any Ethereum specific language in the meat of the cookbook.